src/update-origami-component-list.js

```diff
@@ -4,8 +4,8 @@ const AWS = require("aws-sdk");
 const process = require("process");
 const getAllComponentsAndVersions = require("./modules/getAllComponentsAndVersions");
```

Comment:
I spy some un-normalised file names `getAllComponentsAndVersions` and `getProductionCodeFor`.

Reply:
My regex failed!